routes/movies.routes.js

```diff
@@ -0,0 +1,19 @@
+// starter code in both routes/celebrities.routes.js and routes/movies.routes.js
+const router = require("express").Router();
+
+// all your routes here
+router.get('/movies', (req, res, next) => res.render('routes/index.js')) 
+
+router.get('/movies/create', (req, res, next) => res.render('movies/new-movie'));
+
+router.post('/movies/create', (req, res, next) => {
+    const { id } = req.body;
+    Movie.create({Celebrity})
+    .then((Movie) => {
+        return Movie.create({id})
+        res.redirect('/movies')
+    })
+    .catch((err) => res.render('movies/new-movie'));
```

Comment:
For the create movie method you would need to extract all the information from the form, meaning that it should look something like this:
```
const {title, genre, plot, cast} = req.body;
```